### DISCUSSION TITLE
RECOEDING

### DISCUSSION THREAD

**Discussion by brixxnatt** [ant-media/Ant-Media-Server#6515] [Q&A]
Hello , so when the server record the VOD , the vod comes like this  stream2024-07-10_16-50-26.002.mp4
I need to remove the DOT  , and only keep the  DOT for the file extention for example .mp4 only no other DOT in the file name
how to to that Globaly on AMS

thanks

**Comment by brixxnatt** [comment]
no one here wants to check this ? please

**Comment by burak-58** [comment]
Hi @brixxnatt,
Thank you for asking. I am converting this to a discussion.

You may check 
https://antmedia.io/docs/guides/recording-live-streams/user-defined-scripts/

**Reply by yashtandon113** [reply]
@brixxnatt 

As @burak-58 mentioned, you can use the user defined script.

You can write your own bash script to run when the recording is finished. The script will rename the file and remove the dot from it.
